keps/sig-windows/2258-node-service-log-viewer/kep.yaml

```diff
@@ -24,18 +25,18 @@ stage: alpha
 # The most recent milestone for which work toward delivery of this KEP has been
 # done. This can be the current (upcoming) milestone, if it is being actively
 # worked on.
-latest-milestone: "v1.24"
+latest-milestone: "v1.25"
 
 # The milestone at which this feature was, or is targeted to be, at each stage.
 milestone:
-  alpha: "v1.24"
-  beta: "v1.25"
-  stable: "v1.26"
+  alpha: "v1.25"
+  beta: "v1.26"
```

Comment:
nit - Let's remove the beta/stable milestones for now (to avoid potentially unnecessary updates in the future).